tests/test_smab.py

```
@@ -117,29 +124,33 @@ def _create_actions(
+                    else model_type.cold_start(
+                        dimension=1, base_model_cold_start_kwargs=base_model_cold_start_kwargs
+                    )  # Zooming
                     for action_id, model_type in zip(action_ids, model_types)
-                }
+                }, base_model_cold_start_kwargs
         else:
             if costs is not None:
                 return {
-                    action_id: model_type(models=[Beta()] * n_objectives, cost=cost)
+                    action_id: model_type(models=[Beta(decay_factor=decay_factor)] * n_objectives, cost=cost)
                     for action_id, model_type, cost in zip(action_ids, model_types, costs)
```

Ruff (0.15.17)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)